colcon_bundle/verb/assets/v2_setup.bash

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "$BASH_SOURCE" )" && pwd)"
+
+export PATH="$DIR/usr/sbin:$DIR/usr/bin:$DIR/usr/local/bin:$DIR/sbin:$DIR/bin:$PATH"
+export LD_LIBRARY_PATH="$DIR/lib:$DIR/usr/lib:$DIR/lib/x86_64-linux-gnu:$DIR/usr/lib/x86_64-linux-gnu:$DIR/usr/lib/x86_64-linux-gnu/mesa:$LD_LIBRARY_PATH:$DIR/lib/arm-linux-gnueabihf:$DIR/usr/lib/arm-linux-gnueabihf:$LD_LIBRARY_PATH:$DIR/lib/aarch64-linux-gnu:$DIR/usr/lib/aarch64-linux-gnu"
+
+export LC_ALL=C.UTF-8
+
+if [ -f $DIR/opt/ros/kinetic/setup.sh ]; then
+	if [ -f $DIR/opt/ros/kinetic/setup.bash ]; then
```

Comment:
i'm not sure i understand why the script has so many lines. it kind of seems like you actually mean something like

```
if [ -f $DIR/opt/ros/kinetic/setup.bash ]; then
  _CATKIN_SETUP_DIR=$DIR/opt/ros/kinetic . $DIR/opt/ros/kinetic/setup.bash
elif [ -f $DIR/opt/ros/kinetic/setup.sh ]; then
  _CATKIN_SETUP_DIR=$DIR/opt/ros/kinetic . $DIR/opt/ros/kinetic/setup.sh
elif [ -f $DIR/opt/ros/melodic/setup.bash ]; then
  _CATKIN_SETUP_DIR=$DIR/opt/ros/melodic . $DIR/opt/ros/melodic/setup.bash
elif [ -f $DIR/opt/ros/melodic/setup.sh ]; then
  _CATKIN_SETUP_DIR=$DIR/opt/ros/melodic . $DIR/opt/ros/melodic/setup.sh
# ad nauseum
```

Which is maybe actually something more like

```
for DISTRO in kinetic melodic dashing
do
  _CATKIN_SETUP_DIR=$DIR/opt/ros/$DISTRO
  if [ -f $_CATKIN_SETUP_DIR/setup.bash ]; then
    . $_CATKIN_SETUP_DIR/setup.bash
    break
  elif [ -f $_CATKIN_SETUP_DIR/setup.sh ]; then
    . $_CATKIN_SETUP_DIR/setup.sh
    break
  fi
done
```

Reply:
@emersonknapp This is a copy and paste of the sh one. Why do we need this again? Also you improved it now, but now we have two versions to maintain, so not sure that we're gaining anything here.

Reply:
https://github.com/colcon/colcon-bundle/blob/master/colcon_bundle/verb/assets/v2_setup.sh